### ISSUE TITLE
Support SpecFlow 3.0 when it will be released

### ISSUE THREAD

**Issue by nvborisenko** [reportportal/agent-net-specflow#39]
Breaking changes:
- New configuration in `App.Config` file. Now it is removed at all.
- SpecFlow will discover RP plugin automatically (via dll file name).
- `Initialize` signature changed to `public void Initialize(RuntimePluginEvents runtimePluginEvents, RuntimePluginParameters runtimePluginParameters, UnitTestProviderConfiguration unitTestProviderConfiguration)`
- Plugin should copied to output dicrectory

**Comment by planetary-rasengan** [comment]
We're experimenting with SpecFlow 3 (3.0.188) and observed ReportPortal.SpecFlow 2.0.0-beta.2 is throwing an exception during test execution. Any updates on ReportPortal supporting SpecFlow 3?

```
[3/28/2019 11:26:43 AM Informational] [xUnit.net 00:00:04.71]       System.AggregateException : One or more errors occurred.
[3/28/2019 11:26:43 AM Informational] [xUnit.net 00:00:04.71]       ---- System.MissingMethodException : Method not found: 'Void TechTalk.SpecFlow.Bindings.BindingInvoker..ctor(TechTalk.SpecFlow.Configuration.SpecFlowConfiguration, TechTalk.SpecFlow.ErrorHandling.IErrorProvider)'.
[3/28/2019 11:26:43 AM Informational] [xUnit.net 00:00:04.71]       ---- The following constructor parameters did not have matching fixture data: FixtureData fixtureData
[3/28/2019 11:26:43 AM Informational] [xUnit.net 00:00:04.71]       Stack Trace:
[3/28/2019 11:26:43 AM Informational] [xUnit.net 00:00:04.71]         
[3/28/2019 11:26:43 AM Informational] [xUnit.net 00:00:04.71]         ----- Inner Stack Trace #1 (System.MissingMethodException) -----
[3/28/2019 11:26:43 AM Informational] [xUnit.net 00:00:04.71]            at ReportPortal.SpecFlowPlugin.SafeBindingInvoker..ctor(SpecFlowConfiguration specFlowConfiguration, IErrorProvider errorProvider)
[3/28/2019 11:26:43 AM Informational] [xUnit.net 00:00:04.71]         ----- Inner Stack Trace #2 (Xunit.Sdk.TestClassException) -----
```

**Comment by nvborisenko** [comment]
SpecFlow 3.0.188 is released, ReportPortal.SpecFlow 2.0.0 is released too.

@pete-ryu, the fix of your above issue is included in v2.0.0.
